_posts/2017-03-14-configuration.md

```diff
@@ -31,7 +31,7 @@ public final class OkHttpLibraryGlideModule extends LibraryGlideModule {
 
 Using the [``@GlideModule``][5] annotation requires a dependency on Glide's annotations:
 ```groovy
-annotationProcessor 'com.github.bumptech.glide:annotation:4.0.0-RC0'
+annotationProcessor 'com.github.bumptech.glide:annotations:4.0.0-RC0'
```

Comment:
Sadly these are reversed, annotations should be compile, compiler should annotationProcessor.

These switched back and forth a few times, sorry for the confusion and thanks for fixing this.